plot_types/basic/stackplot.py

```diff
@@ -0,0 +1,28 @@
+"""
+===============================
+stackplot(x, y)
+===============================
+See `~matplotlib.axes.Axes.stackplot`
+"""
+import matplotlib.pyplot as plt
+import numpy as np
+
+plt.style.use('_mpl-gallery')
+
+# make data
+x = np.arange(0, 10, 2)
+ay = [1, 1.25, 2, 2.75, 3]
+by = [1, 1, 1, 1, 1]
+cy = [2, 1, 2, 1, 2]
+y = np.vstack([ay, by, cy])
+labels = ['a', 'b', 'c']
+# plot
+fig, ax = plt.subplots()
+
+ax.stackplot(x, y, labels=labels)
```

Comment:
The current colours are quite stark compared to the rest of this section, though they are the same as the next section, so maybe it is OK?  If it were me, I'd apply an alpha, probably in the cycle?

I clicked through to stackplot and its description of the y parameter is somewhat enigmatic.  I would not have assumed that the third plotted "line" was the cumulatively summed with the first two.